fs/glob.ts

```diff
@@ -0,0 +1,6 @@
+import { FileInfo } from "deno";
+import { globrex, GlobOptions } from "./globrex.ts";
+
+export function glob(glob: string, options: GlobOptions = {}): RegExp {
```

Comment:
In future commits please add jsdocs to exported methods. (With code examples.)

Reply:
For sure i was thinking about it. Have to check with @hayd what's the best place and examples for glob usage. Working on this asap!